teps/0042-taskrun-breakpoint-on-failure.md

```diff
@@ -1,35 +1,40 @@
 ---
-title: taskrun-breakpoint-on-failure
+title: breakpoints-for-taskruns-and-pipelineruns
 authors:
   - "@waveywaves"
 creation-date: 2021-01-15
-last-updated: 2021-03-21
+last-updated: 2021-11-29
 status: proposed
```

Comment:
I noticed that this is marked as `proposed` while it should have been marked as `implemented`.
I think it would be ideal to update this one as implemented and create a new TEP for the new features.

Reply:
Marking this as implemented :heavy_check_mark: